pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -22,12 +22,16 @@ import (
 	"io"
 	"net"
 	"net/http"
+	ossys "os"
 	"regexp"
+	"strings"
 	"time"
 
 	"code.google.com/p/gcfg"
 	"github.com/rackspace/gophercloud"
 	"github.com/rackspace/gophercloud/openstack"
+	"github.com/rackspace/gophercloud/openstack/blockstorage/v1/volumes"
+	"github.com/rackspace/gophercloud/openstack/compute/v2/extensions/volumeattach"
```

Comment:
We should add a test to hack/test-cmd.sh that verifies that testing flags are not linked in to any of our binaries, since apparently keeping them out is one of the Hard Problems.